conda/recipes/jupyterlab-nvdashboard/meta.yaml

```diff
@@ -17,18 +17,19 @@ build:
 
 requirements:
   build:
-    - python >=3.5
+    - python >=3.6
     - setuptools
     - pynvml
     - psutil
+    - nodejs
+    - jupyter-packaging >=0.7.0,<0.8
   run:
-    - python >=3.5
-    - jupyterlab >=2.0
-    - jupyter-server-proxy
+    - python >=3.6
+    - jupyterlab >=3.0.0
+    - jupyter-server-proxy >=1.3.2
     - bokeh >2.1
     - pynvml
     - psutil
-    - nodejs
 
 
 test:
```

Comment:
Also we may want [some file tests like these]( https://github.com/conda-forge/jupytext-feedstock/pull/54/files#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR39-R42 )

Reply:
Thanks, added in 255fa71